### PR TITLE
convert from netlify to go

### PR DESCRIPTION
**Pull request by rybit**
mostly just a lot of search and replace ~ @biilmann could you check this is right?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
-# Netlify Commerce JS
+# Go Commerce JS
```
Comment (verythorough) [inline README.md]: Remove space

File: README.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 
-This is a JS client library for [Netlify Commerce](https://github.com/netlify/netlify-commerce) API.
+This is a JS client library for [Go Commerce](https://github.com/netlify/gocommerce) API.
```
Comment (verythorough) [inline README.md]: Remove space

File: README.md [hunk 3]
```diff
@@ -80,7 +80,7 @@ commerce.setCountry("USA");
 commerce.setCurrency("USD");
 ```
 
-You can use Netlify Commerce JS together with [authlify](https://github.com/netlify/authlify) to let users log in and claim view order history.
+You can use Go Commerce JS together with [GoTrue](https://github.com/netlify/gotrue) to let users log in and claim view order history.
```
Comment (verythorough) [inline README.md]: Remove space

**Review by verythorough — CHANGES_REQUESTED** [review]
Found a few instances of "Go Commerce" with an extra space. I don't have commit access on this repo, so I went the easy way and just made comments. 😈

**Comment by verythorough** [comment]
Looks good to me! Though I can't fully confirm the code part is good. I assume npm/etc. handles the name change so projects using netlify-commerce-js don't break?

**Comment by bdougie** [comment]
Authlify needs to be changed to goTrue in the readme

**Comment by bdougie** [comment]
also src/index.js references `NetlifyCommerce` and `netlify.commerce`

**Comment by rybit** [comment]
I am not seeing those @bdougie

**Comment by bdougie** [comment]
my bad, I was looking at master. This is good

**Pull request merged**